locator/topology.hh

```diff
@@ -252,6 +252,12 @@ public:
         return _dc_endpoints;
     }
 
+    const std::unordered_map<sstring,
+                            std::unordered_set<const node*>>&
```

Comment:
(we should have used unordered_multimap)

Reply:
do you want me to fix it now or can come later? perhaps a FIXME?

Reply:
one good reason to keep it as is, is to avoid duplicated nodes. unordered_multimap allows multiple copies of same element. of course, we could adjust it during insertion, to disallow multiple copies, but with today's type, we get it off-the-shelf.

Reply:
Certainly not for now.